src/components/DigitalCredentials/CredentialsLanding.vue

```diff
@@ -1,5 +1,4 @@
 <template>
-  <!-- Digital Credentials Landing -->
```

Comment:
This caused a parsing issue (probably because only 1 top level element is allowed).